.gitignore

```diff
@@ -0,0 +1 @@
+generated
```

Comment:
This is not related to https://github.com/siadat/blgo/issues/3 but is simple and good to have.